integration-tests/utils/integration-helpers.js

```diff
@@ -170,10 +172,25 @@ const withRegisteredUser = test => withUser(async options => {
   await test(options)
 })
 
+const enableDataCatalog = async (page, testUrl, token) => {
+  await page.goto(testUrl)
+  await waitForNoSpinners(page)
+
+  await findText(page, 'Browse Data')
+  await page.evaluate(() => window.configOverridesStore.set({ isDataBrowserVisible: true }))
+  await page.reload({ waitUntil: ['networkidle0', 'domcontentloaded'] })
```

Comment:
@dmohs had made a comment in a previous PR about throwing an error when we no longer need this flag to ensure it gets cleaned up. I thought that is a good idea, so created [this ticket](https://broadworkbench.atlassian.net/browse/DC-176) to track this. This looks like a good step in that direction.

I would prefer not making any changes for this PR because I think it is a good iterative improvement. We can follow up with changing how we handle the feature flag in a small future PR.

Also, once the flag handling has been adjusted, we can make this function more generic to apply it to other tests that share some of this boilerplate. I looked at some of the existing tests, it looks like this helper function would be of good use in some of those as well.